bigbluebutton-html5/imports/ui/components/app/styles.scss

```diff
@@ -150,12 +150,31 @@ $bars-padding: $lg-padding-x - .45rem; // -.45 so user-list and chat title is al
 .media {
   @extend %full-page;
   flex: 1 100%;
-  order: 2;
+
+  flex-direction: row;
+  position: relative;
 
   @include mq($small-only) {
     padding-bottom: $actionsbar-height;
     margin-bottom: $actionsbar-height;
-    position: relative;
+  }
+
+  @include mq($portrait) {
+    flex-direction: column;
+  }
+}
+
+.closedCaptionBox {
+  position: relative;
+  flex-basis: 12%;
```

Comment:
Could you increase the size of this box, please? Sorry for confusing you, let's make it `15vw`.
I found that it's a bit small and it shrinks even more when you open a user list and chat if we use percentages.